### PR TITLE
add missing dependencies for libheif (libpng, libjpeg-turbo) + switch to configuring build with CMake so libde265 dependency is picked up

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by jfgrimm** [comment]
Test report by @jfgrimm
**FAILED**
Build succeeded for 3 out of 4 (1 easyconfigs in total)
node134.pri.viking.alces.network - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6138 CPU @ 2.00GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/1498c6cf94628bdd9d67d9a7d2496239 for a full test report.

**Comment by jfgrimm** [comment]
```
9409 CMake Error at gdk-pixbuf/cmake_install.cmake:60 (file):                                                                
9410   file INSTALL cannot copy file                                                                                         
9411   "/tmp/jfg508/libheif/1.12.0/GCC-11.2.0/easybuild_obj/gdk-pixbuf/libpixbufloader-heif.so"                              
9412   to "/usr/lib64/gdk-pixbuf-2.0/2.10.0/loaders/libpixbufloader-heif.so":                                                
9413   Permission denied.                                                                                                    
9414 Call Stack (most recent call first):                                                                                    
9415   cmake_install.cmake:53 (include)
```

File: easybuild/easyconfigs/l/libheif/libheif-1.12.0-GCC-11.2.0.eb [hunk 1]
```diff
@@ -12,14 +12,26 @@ source_urls = ['https://github.com/strukturag/libheif/releases/download/v%(versi
...
+builddependencies = [
+    ('CMake', '3.22.1'),
+]
+
 dependencies = [
+    ('libpng', '1.6.37'),
+    ('libjpeg-turbo', '2.0.6'),
     ('libde265', '1.0.8'),
     ('x265', '3.5'),
 ]
```
Comment (jfgrimm) [inline easybuild/easyconfigs/l/libheif/libheif-1.12.0-GCC-11.2.0.eb:24]:
```suggestion
    ('Gdk-Pixbuf', '2.42.6'),
]
```

**Review by jfgrimm — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/l/libheif/libheif-1.12.0-GCC-11.2.0.eb (hunk 1 above)
Comment (jfgrimm) [reply easybuild/easyconfigs/l/libheif/libheif-1.12.0-GCC-11.2.0.eb:24]: build succeeds for me when `Gdk-Pixbuf` is added

Comment (verdurin) [reply easybuild/easyconfigs/l/libheif/libheif-1.12.0-GCC-11.2.0.eb:24]: @boegel ping?

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node2619.swalot.os - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/ce1e8117e1a2a5ef5f6a0656b7ece5bf for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3100.skitty.os - Linux RHEL 8.4, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/e50b604c4302c2086e8b8c48b3e7bd19 for a full test report.

File: easybuild/easyconfigs/l/libheif/libheif-1.12.0-GCC-11.2.0.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/l/libheif/libheif-1.12.0-GCC-11.2.0.eb:25]: @verdurin fixed with 60bfb0c

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/c32c58a0521a6fd84db898e0e5da0690 for a full test report.

**Review by verdurin — APPROVED** [review]
Looks fine.

**Comment by verdurin** [comment]
@jfgrimm could you confirm you're happy with the fix?

**Review by jfgrimm — APPROVED** [review]

**Comment by verdurin** [comment]
Going in, thanks @boegel!

**Pull request merged**